### DISCUSSION TITLE
Use `mount` method from trait in Volt component

### DISCUSSION THREAD

**Discussion by jacksleight** [livewire/livewire#6158] [1. Help]
I'm using a trait in a Volt component and I'd like to define a `mount` method in that trait that will be used by all Volt components that use that trait, rather than defining it for every component with the Volt mount function.

However, at the moment the trait's mount method is not being called, even though I'm not using Volt's mount function.

Is there a way to make this work?

**Comment by joshhanley** [comment]
@jacksleight can I suggest opening this up on the Volt repo?

**Reply by jacksleight** [reply]
Sure thing will do. I wasn't too sure on the best place to post Volt questions.

**Reply by joshhanley** [reply]
No worries! 🙂
